content/concepts/tokens/clawback-of-issued-currency.md

```diff
@@ -0,0 +1,36 @@
+---
+html: clawback-of-issued-currency.html
+parent: tokens.html
+blurb: Issuers can claw back their issued tokens for compliance purposes.
+labels:
+  - Tokens
+status: not_enabled
+---
+# Clawback
+<embed src="/snippets/_clawback-disclaimer.md" />
+For regulatory purposes, some issuers must have the ability to recover issued tokens after they are distributed to accounts. For example, if an issuer were to discover that tokens were sent to an account sanctioned for illegal activity, the issuer could recover, or *claw back*, the funds.
+
+Issuers can create tokens that they can claw back by adding the `lsfAllowTrustLineClawback` flag to the issuing account.
+
+**Note:** You can only claw back issued tokens created by your account. You cannot claw back XRP in this way.
+
+Clawback is disabled by default. To use clawback, you must set the `lsfAllowTrustLineClawback` flag through an `AccountSet` transaction. The `AccountSet` transaction only succeeds if the account has an empty owner directory, meaning that the account has no trust lines, offers, escrows, payment channels, or checks. After you set this flag, it cannot reverted. The account permanently gains the ability to claw back issued assets on trust lines.
```

Comment:
```suggestion
Clawback is disabled by default. To use clawback, you must send an [AccountSet transaction][] to enable the **Allow Clawback** setting. **An issuer with any existing tokens cannot enable Clawback.** You can only enable Allow Clawback if you have a completely empty owner directory, meaning you must do so before you set up any trust lines, offers, escrows, payment channels, checks, or even signer lists.
```

Note, the reference-style link I use for AccountSet transaction won't work unless you include the common links snippets at the end of the file.

Reply:
Thank you, Rome, for your thorough review.